css/style.css

```diff
@@ -0,0 +1,402 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: system-ui, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  background-color: #f9f9f9;
+}
+
+nav {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  color: #036bd3;
+}
+
+button {
+  margin-top: 0.5rem;
+  height: 3rem;
+  width: 11rem;
+  border: 1px solid #036bd3;
+  color: #036bd3;
+  font-size: 10px;
+  border-radius: 0.8rem;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+header {
+  background-image: linear-gradient(115deg, #1a1a1a80, #0d0d0e70), url("../assets/her-one.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 0 0.2em 2rem;
+
+  i {
+    color: #036bd3;
+    cursor: pointer;
+  }
+
+  button {
+    border: 2px solid #f9f9f9;
+    background-color: transparent;
+  }
+
+  a {
+    color: #f9f9f9;
+  }
+}
+
+h2 {
+  font-family: "Courier New", Courier, monospace;
```

Comment:
I see this is the second time you are using this.

- [ ] use CSS variable to declare this font-family as either the primary or secondary font family, So you don't have to write it always